kernel/tests/write.rs

```diff
@@ -704,3 +705,84 @@ async fn test_append_invalid_schema() -> Result<(), Box<dyn std::error::Error>>
     }
     Ok(())
 }
+
+#[tokio::test]
+async fn test_idempotent_writes() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Nit: this is really just `test_write_appids`

Reply:
changed to `test_write_txn_actions`